student/src/main/java/tds/student/services/AccommodationsService.java

```diff
@@ -397,7 +397,7 @@ public int compare (IGrouping<AccommodationType, AccommodationValue> o1, IGroupi
     // if this is PT then remove all acc's that are disabled for guest sessions
     try {
       if (isGuestSession)
-        CollectionUtils.removeAll (accList, CollectionUtils.select (accList, new Predicate ()
+        CollectionUtils.removeAll (accList.getData(), CollectionUtils.select (accList.getData(), new Predicate ()
```

Comment:
This is fallout from changing AccList from `extends ArrayList<Data>` to containing an `ArrayList<Data>` property.